easy/admin/field.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, attr, short_description=None, admin_order_field=None, allow_t
 
         super(SimpleAdminField, self).__init__(short_description, admin_order_field, allow_tags)
 
+    def __name__(self):
+        return 'SimpleAdminField'
```

Comment:
Can you send me the warning message from django?

Reply:
```
'ForeignKeyAdminField' object has no attribute '__name__'
```

When used like this

```
price_model_link = ForeignKeyAdminField('price_model')

readonly_fields = (
        price_model_link,
)
```

Reply:
What do you think about put this way:

```python
    def __name__(self):
        return self.__class__.__name__
```

It will work for all child classes with right name.